cadc-log/src/main/java/ca/nrc/cadc/log/LogControlServlet.java

```diff
@@ -359,29 +371,173 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
     /**
      * Check for proper group membership.
      */
-    private void authorize(HttpServletRequest request, boolean readOnly) throws AccessControlException, TransientException {
-        if (authorizerClassName != null && groupAuthorizer == null) {
-            throw new RuntimeException("CONFIG: group authorizer was configured but failed to load: " + authorizerClassName);
+    private void authorize(HttpServletRequest request, boolean readOnly)
+        throws AccessControlException, TransientException {
+
```

Comment:
The start of this method should create a Subject for the caller and pass that into other methods. That's something that has to happen once so should be here. You added an optional lazy augment to AuthenticationUtil -- use it here.

Reply:
This is the comment that relates to getPrincipals(X500Principal.class) above

Reply:
Changed to not augment the Subject.